module/scripts/combat/effects.js

```diff
@@ -0,0 +1,94 @@
+// module\scripts\combat\effects.js
+export { applyBleed, applyFire, applyPoison, applySuffocation };
+
+async function changeHealth(actor, amount, name, remidy = "") {
+    // Ensure `amount` is a number
+    if (typeof amount !== 'number' || !isFinite(amount)) {
+      console.error(`Invalid amount for health change: ${amount}`);
+      return;
+    }
+  
+    const currentHP = actor.system.derivedStats.hp.value;
+    const maxHP = actor.system.derivedStats.hp.max;
+    const newHP = Math.min(Math.max(currentHP + amount, 0), maxHP);
+  
+    // Ensure the new HP is within valid range
+    if (newHP !== currentHP) {
+      await actor.update({
+        'system.derivedStats.hp.value': newHP
+      });
+  
+      if (amount > 0) {
+        ChatMessage.create({
+          content: `${actor.name} heals for ${amount} hit points because of ${name}!${remidy ? ` ${remidy}` : ''}`,
+          speaker: ChatMessage.getSpeaker({ actor: actor })
+        });
+      } else if (amount < 0) {
+        ChatMessage.create({
+          content: `${actor.name} takes ${-amount} hit points of ${name} damage!`,
+          speaker: ChatMessage.getSpeaker({ actor: actor })
+        });
+      }
+    } else {
+      console.log(`No change in HP detected for ${actor.name}.`);
+    }
+  }
+  
+  async function applyFireDamageToEquippedItems(actor, damage) {
```

Comment:
This needs to stay here

Reply:
I am unsure how I would check if the damage was already applied through applyDamageToAllLocations if I use that function.

Reply:
You dont need to check it. The reliability/SP damage is an additional effect